src/Initializers/Migrations.js

```diff
@@ -0,0 +1,19 @@
+class Migrations {
```

Comment:
Is there a way to ensure this runs first before anything has a chance to touch the DB?

Reply:
Sure.